package.json

```diff
@@ -5,6 +5,7 @@
     "node-gyp": "^3.6.2",
     "nunjucks": "^3.0.1",
     "typescript": "^2.4.2",
-    "webidl2": "^4.1.0"
+    "webidl2": "^4.1.0",
+    "mocha": "^3.5.0"
```

Comment:
Please keep this in alphabetical order. :)

Reply:
done.